game/game/playercontrol.cpp

```diff
@@ -905,8 +905,8 @@ void PlayerControl::processPickLock(Npc& pl, Interactive& inter, KeyCodec::Actio
 
   if(pickLockProgress<cmp.size() && std::toupper(cmp[pickLockProgress])!=ch) {
     pickLockProgress = 0;
-    const int32_t dex = pl.attribute(ATR_DEXTERITY);
-    if(dex<int32_t(script.rand(100)))  {
+    const int32_t dex = Gothic::inst().version().game==2 ? pl.attribute(ATR_DEXTERITY) : 100 - pl.talentValue(TALENT_PICKLOCK);
+    if(dex<=int32_t(script.rand(100)))  {
```

Comment:
Why `<=`? Rand(100) returns number in [0..99] range. That means, if dexterity equals 99 player will always be able to to open locks. Is that intended?

Reply:
This happens with `<`.  `99<=99` enters the delete branch and 100 is required to never loose a lockpick.

Reply:
Please note those kind of changes explicitly in future PR's - this will make review process more simple